### DISCUSSION TITLE
UTC datetime(2023-04-28 17:27:19) to 'fff' format('April 28, 2023 at 12:27 PM CDT') and then convert back to original

### DISCUSSION THREAD

**Discussion by manjindersinghsarkaria** [moment/luxon#1501] [General]
I have a UTC datetime (2023-04-28 17:27:19), which I converted into the 'fff' format, appearing as ('April 28, 2023 at 12:27 PM CDT'). Now, I am looking for a way to convert 'April 28, 2023 at 12:27 PM CDT' back to the original format, '2023-04-28 17:27:19'.

DateTime.fromSQL('2023-04-28 17:27:19', { zone: 'UTC' })
    .setZone('local')
    .toFormat("fff")
    
    This gives **April 28, 2023 at 12:27 PM CDT** but I am not sure how can go back to original date i passed which is **2023-04-28 17:27:19**

**Comment by diesieben07** [comment]
So you want to parse the string `April 28, 2023 at 12:27 PM CDT` into a `DateTime`, correct? This is unfortunately not possible, because shortened time zone names (like CDT in your example) are ambiguous. Even if they were not ambiguous, there isn't an API available in the browser to convert the string `CDT` into a usable time zone.
This is explained in the [section about parsing limitations in the documentation](https://moment.github.io/luxon/#/parsing?id=limitations).
